homeworks/kutwicki_serge/HW02/calculation.py

```diff
@@ -0,0 +1,24 @@
+"""Base math functions for the HW02."""
+
+
+def add(num1, num2):
+    """Add two numbers"""
+    return num1 + num2
+
+
+def subtract(num1, num2):
+    """Subtract two numbers"""
+    return num1 - num2
+
+
+def multiply(num1, num2):
+    """Multiply two numbers"""
+    return num1 * num2
+
+
+def divide(num1, num2):
+    """Divide two numbers"""
+    try:
+        return num1 / num2
+    except ZeroDivisionError:
+        return -1
```

Comment:
@Tempos Why did you put this try-except here not int the 'test_divide'?

Reply:
@ProffesorRabbit ZeroDivision check wasn't mentioned/requested in this task (only negative value check), so I've decided to put it in next logical place.

Reply:
@Tempos please read last part of task again 

> A test has to fail if the calculation result is less then 0, otherwise, the test is passed.

Reply:
@ProffesorRabbit It woud log as ERROR, but with exception check and returning -1, test passes successfuly.